home/private_dot_local/lib/agent-equipment/agent_equipment/execution_authority.py

```
@@ -0,0 +1,2401 @@
+        return recovery == {"kind": "none", "reason": "absent_noop"}
+    return recovery == {
+        "kind": "none",
+        "reason": (
+            "operator_owned"
+            if route.get("control_owner") == "operator_owned"
+            else "already_desired"
+        ),
+    }
+
+
+def _captured_action_contexts(
```

Ruff (0.16.2)

[warning] Too many return statements (39 > 6)

(PLR0911)